apps/web-mzima-client/src/app/settings/categories/create/create.component.ts

```diff
@@ -21,6 +28,11 @@ export class CreateComponent {
         this.router.navigate(['settings/categories']);
       },
       error: ({ error }) => {
+        if (error.errors[0].status === 404) {
+          this.notificationService.showError(
+            this.translate.instant('An error occured, please try again'),
+          );
```

Comment:
Is there a way to show the error-message as well, for example, look at how its handled here: https://github.com/ushahidi/platform-client-mzima/blob/development/apps/web-mzima-client/src/app/post/post-edit/post-edit.component.ts#L644